test/integration/conftest.py

```diff
@@ -300,3 +301,46 @@ def create_vpc_with_subnet_and_linode(get_client, create_vpc_with_subnet):
     yield vpc, subnet, instance, password
 
     instance.delete()
+
+
+@pytest.fixture(scope="session")
```

Comment:
I think the scope is not session? The new VPC will need to be created every time this fixture is accessed, right? Or not?

Reply:
Test cases that exist right now can work with this one session vpc create instance. We can change this as we add more tests that require different instances to be created